apps/bfd-server/bfd-server-shared-utils/src/main/java/gov/cms/bfd/server/sharedutils/MDC.java

```diff
@@ -0,0 +1,76 @@
+package gov.cms.bfd.server.sharedutils;
+
+import org.slf4j.spi.MDCAdapter;
+
+/**
+ * Wrapper for to the {@link org.slf4j.MDC} class (used in logging). Historically, we have used "."
+ * to delimit parts of the MDC keys, such as "http_access.request.header". For AWS CloudWatch
+ * Metrics, though, the "." character is not supported, so we need to change these to "_".
+ */
+public class MDC {
```

Comment:
I'm going to review a little more carefully tomorrow but I recommend changing the name of this to something else, like BfdMDC or MDCWrapper to minimize surprises since most will assume that the MDC in the code comes from slf4j